test/k8s/deploy.yaml

```diff
@@ -24,4 +24,8 @@ spec:
       containers:
       - name: demo
         image: ghcr.io/containerd/runwasi/wasi-demo-app:latest
-        imagePullPolicy: Never
\ No newline at end of file
+        imagePullPolicy: Never
+      - name: nginx
```

Comment:
does this work on k3s?

Reply:
`make test/k3s` uses WasmEdge shim only, which I haven't implemented DefaultExecutor to yet. So the answer is no, it doesn't work, until we implement the same thing for wasmedge shim.

Reply:
I have updated this PR to include nginx on k3s.